Makefile.dep

```diff
@@ -994,6 +994,10 @@ ifneq (,$(filter periph_%, $(USEMODULE)))
   USEMODULE += periph_common
 endif
 
+ifneq (,$(filter periph_uart_nonblocking, $(USEMODULE)))
+  USEMODULE += periph_uart
+endif
+
```

Comment:
Why is this needed ?

Reply:
So everything that depends on `periph_uart` is also fulfilled with `periph_uart_nonblocking`

Reply:
But should it belongs to Makefile.dep ? Why not cpu/sam0_common/Makefile.dep ?

Reply:
So other platforms can implement it too.